CONTRIBUTING.md

```diff
@@ -185,31 +295,55 @@ cached.
     ```
 
 4. After building the documentation, open the local build and examine the
-examples gallery for any obvious issues. If issues are present, abort the
-release.
+examples gallery for any obvious issues.
+
+5. Update the version numbers in `pyvista/_version.py` and commit it.
+   Push the branch to GitHub and create a new PR for this release that
+   merges it to master.  Development to master should be limited at
+   this point while effort is focused on the release.
+
+6. It is now the responsibility of the `pyvista` community to
+   functionally test the new release.  It is best to locally install
+   this branch and use it in production.  Any bugs identified should
+   have their hotfixes pushed to this release branch.
 
+7. When the branch is deemed as stable for public release, the PR will
+   be merged to master and the `master` branch will be tagged with a
+   `MAJOR.MINOR.0` release.  The release branch will not be deleted.
```

Comment:
Tiny question here: so the next minor release will be `0.25.0`?

Reply:
Correct.